msfrpcd

```diff
@@ -4,52 +4,161 @@
 # $Id$
 #
 # This user interface listens on a port and provides clients that connect to
-# it with an RPC interface to the Metasploit Framework.
+# it with an RPC or JSON-RPC interface to the Metasploit Framework.
 #
 # $Revision$
 #
 
-msfbase = __FILE__
-while File.symlink?(msfbase)
-  msfbase = File.expand_path(File.readlink(msfbase), File.dirname(msfbase))
+RPC_TYPE = 'Msg'
+
+@localconf = "#{ENV['HOME']}/.msf4"
```

Comment:
The name of this location could change. Is there any way we can pull it from framework directly?

Reply:
```
msf5 > pry
[*] Starting Pry shell...
[*] You are in the "framework" object

[1] pry(#<Msf::Framework>)> ::Msf::Config.config_directory
=> "/root/.msf4"
[2] pry(#<Msf::Framework>)> 
```